src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -154,9 +154,6 @@
 
     <!-- Need to be passed in as an env var so that custom Exec tasks in the repo's DotNetBuild.props receive this setting. -->
     <EnvironmentVariables Include="DotNetPackageVersionPropsPath=$(PackageVersionPropsPath)" />
-
-    <!-- Needed for miscellanous projects in various repos - see https://github.com/dotnet/source-build/issues/4081-->
```

Comment:
What has changed to address https://github.com/dotnet/source-build/issues/4081?

Reply:
Nothing yet. This PR was essentially to go find the places that need alteration.